internal/backup_push_handler.go

```diff
@@ -128,7 +128,7 @@ func createAndPushBackup(
 	tracelog.ErrorLogger.FatalOnError(err)
 	err = bundle.FinishQueue()
 	tracelog.ErrorLogger.FatalOnError(err)
-	uncompressedSize := bundle.TarBall.Size()
+	uncompressedSize := atomic.LoadInt64(bundle.AllTarballsSize)
 	compressedSize := atomic.LoadInt64(uploader.tarSize)
```

Comment:
These two size assignments are executed before uploadPGControl and uploadlabelfiles tars, I think they should be assigned after.

Reply:
fixed.